presto-main/src/main/java/com/facebook/presto/sql/relational/SqlToRowExpressionTranslator.java

```diff
@@ -341,6 +342,23 @@ protected RowExpression visitSymbolReference(SymbolReference node, Void context)
             return new VariableReferenceExpression(node.getName(), getType(node));
         }
 
+        @Override
+        protected RowExpression visitDynamicFilterExpression(DynamicFilterExpression node, Void context)
+        {
+            // Hack!
```

Comment:
this class doesn't have much to do with predicate pushdown.

Just rewrite to:
```
constant(true, BOOLEAN);
```
for now.